addons/mail/wizard/mail_compose_message.py

```diff
@@ -231,7 +233,11 @@ def _compute_body(self):
         template, reset it. """
         for composer in self:
             if composer.template_id:
-                composer._set_value_from_template('body_html', 'body')
+                try:
+                    composer._set_value_from_template('body_html', 'body')
```

Comment:
So it's an issue with qweb? Wouldn't it make more sense to wrap the actual _generate_template call if fixing it here?